automated install/basic-install.sh

```diff
@@ -1265,6 +1265,25 @@ check_service_active() {
   fi
 }
 
+# Systemd-resolved's DNSStubListener and dnsmasq can't share port 53. 
+# Resolved needs to remain in place for installer to download needed files
+# so this change needs to be made after installation is complete, but before resarting dnsmasq/ftl
+disable_resolved_stublistener() {
+  # Check if Systemd-resolved's DNSStubListener is enabled and active on port 53
+  if check_service_active "systemd-resolved"; then
+    # Check if DNSStubListener is enabled
+    if ( grep -E '#?DNSStubListener=yes' /etc/systemd/resolved.conf &> /dev/null ); then
+      # Disable the DNSStubListener to unbind it from port 53
+      # Note that this breaks dns functionality on host until dnsmasq/ftl are up and running
+      echo -e "Disabling systemd-resolved DNSStubListener"
```

Comment:
@PromoFaux @WaLLy3K Is this in agreement with your `${TICK}` magic in the installer?

Reply:
It'll stick out like a sore thumb if the tick/info/cross stuff isn't included, that's for sure!